### PR TITLE
New stacked content selector

### PR DESCRIPTION
**Pull request by sumeetsb**
Added another version of stacked ws-content-selector directive.

It takes a configuration attribute of stack-type="2" to be activated. The options will have a green background, with checkmark to the left when selected. There is an info icon present when an option has body text available. The info icon is clickable to reveal the body text, otherwise the rest of the option is clickable to select it.

NOTE: Info icon currently isn't changing color on hover and select
screen shot 2017-05-03 at 4 45 16 pm
screen shot 2017-05-03 at 4 45 29 pm
screen shot 2017-05-03 at 4 45 42 pm

### CODE REVIEW

**Comment by benjyblack** [comment]
Can we call this something like `expandable` instead of `stackedV2`?

```html
expandable="true"
```

instead of
```html
stack-type="2"
```

File: index.html [hunk 1]
```diff
@@ -309,6 +310,10 @@
         <ws-content-selector options="stackedContentOptions" selected="selectedStackedOption" type="single" stacked="true"></ws-content-selector>
         Selected Value: {{ selectedStackedOption.value }}
         <hr/>
+        <div>Stacked select version 2</div>
```
Comment (benjyblack) [inline index.html]: Can you change this reference and the one below to be `expandable`

File: src/form/content-selector/_ws-content-selector.scss [hunk 2]
```diff
@@ -30,7 +102,7 @@
     background-color: $color-gray-10;
     position:         relative;
     border-left:      2px solid transparent;
-    box-shadow: 0 1px 2px 0 rgba(0,0,0,0.05);
+    box-shadow: 0 1px 2px 0 rgba(0,0,0,0.05);ggsdgsdgsdfdfsdfsdfvfcx  cb
```
Comment (benjyblack) [inline src/form/content-selector/_ws-content-selector.scss]: Interesting CSS :P

File: src/form/content-selector/ws-content-selector.html [hunk 3]
```diff
@@ -1,17 +1,18 @@
...
     <div ng-repeat="option in row"
          class="option col-{{colSize}}"
-         ng-class="{selected: selectedOptions.indexOf(option) !== -1, disabled: option.disabled, 'no-body': !option.body }"
+         ng-class="{selected: selectedOptions.indexOf(option) !== -1, disabled: option.disabled, 'no-body': !option.body, info: option.moreInfo || !expandable}"
          ng-style="getOptionStyle(option)"
          ng-click="select(option)"
          ng-mouseover="mouseEnter(option)"
          ng-mouseleave="mouseLeave(option)">
-      <strong>{{ option.header }}</strong>
+        <div><strong>{{ option.header }}</strong> <span class="blurb">{{ option.blurb }}</span></div>
```
Comment (benjyblack) [inline src/form/content-selector/ws-content-selector.html]: The elements should be on separate lines for readability

File: src/form/content-selector/ws-content-selector.js [hunk 4]
```diff
@@ -3,7 +3,7 @@
 
   angular
   .module('ws.common')
-  .directive('wsContentSelector', ['$templateCache', '$sce', function ($templateCache, $sce) {
+  .directive('wsContentSelector', ['$templateCache', '$sce', '$window', function ($templateCache, $sce, $window) {
```
Comment (benjyblack) [inline src/form/content-selector/ws-content-selector.js]: Don't need `$window`

File: src/form/content-selector/_ws-content-selector.scss (hunk 2 above)
Comment (sumeetsb) [inline src/form/content-selector/_ws-content-selector.scss]: Uuuuuuhhh...

**Pull request merged**